src/makers/midas/index.ts

```diff
@@ -0,0 +1,9 @@
+import antennas from './antennas'
+import hotspots from './hotspots'
+
+export default {
+  antennas,
+  hotspots,
+  id: 26, 
```

Comment:
Please fix the lint error:

```
  7:10  error  Delete `·`  prettier/prettier
```

Reply:
already deleted the space after "id: 26,"